DotnetCLIVersion.txt

```diff
@@ -1 +1 @@
-1.0.0-beta-002228
```

Comment:
This is one week old cli. The live cli has the support for native compilation disabled. We will need to fix #1075 to pick it up.